starport/services/scaffolder/init.go

```diff
@@ -103,18 +105,18 @@ func (s *Scaffolder) protoc(projectPath, gomodPath string, version cosmosver.Maj
 	var (
 		protoPath    = filepath.Join(projectPath, conf.Build.Proto.Path)
 		includePaths = xos.PrefixPathToList(conf.Build.Proto.ThirdPartyPaths, projectPath)
-		targets      = []cosmosprotoc.Target{
-			cosmosprotoc.WithGoGeneration(gomodPath),
+		targets      = []cosmosproto.Target{
+			cosmosproto.WithGoGeneration(gomodPath),
 		}
 	)
 
-	frontendPath := filepath.Join(projectPath, conf.Frontend.Path)
-	if _, err := os.Stat(frontendPath); err == nil && conf.Build.Proto.JS.Out != "" {
-		path := filepath.Join(projectPath, conf.Build.Proto.JS.Out)
-		targets = append(targets, cosmosprotoc.WithJSGeneration(path))
+	if conf.Client.Vuex.Path != "" {
```

Comment:
Should add one comment to explain this part

Reply:
done by c112e16.